docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -519,7 +625,7 @@ spring:
   cloud:
     azure:
       servicebus:
-        connection-string: [servicebus-namespace-connection-string]
+        connection-string: ${CONNECTION_STRING}
```

Comment:
let's remove this yaml and only write a sentense to give a sample of how to configure the property of spring.cloud.stream.default.producer.partitionKeyExpression.